dashboard/src/pages/TreeDetails/Tabs/Build/BuildTab.tsx

```diff
@@ -191,7 +336,19 @@ const BuildTab = ({ treeDetailsData }: BuildTab): JSX.Element => {
 
   return (
     <div className="flex flex-col gap-8 pt-4">
-      <CardsGroup cards={cards} />
+      <div className="columns-2">
+        <MemoizedStatusCard
+          toggleFilterBySection={toggleFilterBySection}
+          treeDetailsData={treeDetailsData}
+        />
+        {/* TODO Kill the CardGroups component once and for all */}
```

Comment:
also, this chart is loading before the others, we need to consolidate the loading behavior

Reply:
if it is a separated query it should have it own loading, global loadings are very painful to do in react since it is entirely modular